CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.4.1 - 2017-08-08
+
+## Enhancements
+
+- Adds support for Fury 3.0.0-beta.4.
```

Comment:
Please change this line to resemble something like https://github.com/apiaryio/fury-adapter-apib-parser/pull/17/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR3. Also, this is not an enhancement.